apps/linode-marketplace-code-server/roles/code-server/tasks/main.yml

```
@@ -0,0 +1,81 @@
+---
+- name: install required packages
+  apt:
+    pkg:
+      - nginx
+      - curl
+      - git
+    state: latest
+    update_cache: true
+
+# code-server
+- name: download code-server .deb package
+  get_url:
+    url: "{{ code_download_link }}"
```

```suggestion
    url: "{{ code_server_download_link }}"
```
Just so that we can keep the semantics the same